wix-bookings/guides/Terminology.md

```diff
@@ -1,58 +1,57 @@
-| Visitor | Anyone who isn't registered as a member or hasn't logged in to the Wix site. If a visitor creates a booking they're added as a contact. | 
-| `WAITING_LIST` status (Booking) | An indication that the booking is on a waiting list. | 
-| Waitlist | A list of participants that requested to be added to a session that is fully booked, if there should be a cancelation. | 
-| Waitlist time window | How much time a customer has to book a place in the session, before it is offered to the next customer on the waitlist. | 
-| Membership plans with [Wix Pricing Plans](https://support.wix.com/en/article/wix-bookings-about-wix-bookings#selling-membership-plans-and-packages) | A pre-paid bundle of services or a membership that provides access to certain services. | 
\ No newline at end of file
+| Double booking | A double booking occurs when a customer attempts to book a service that's already reserved. This can happen if multiple customers are checking out simultaneously or if the business owner manually confirms more bookings than the service's capacity allows. |
+| Extended booking | In addition to basic information about the booking, an extended booking contains additional details. For example, attendance details, and whether the customer can demand a refund or reschedule the booking. |
+| External calendar | A calendar hosted by a 3rd-party provider. |
+| External calendar connection | A linkage between a Wix Bookings calendar and one or more external calendars. External calendar connections allow you to import and export calendar events. |
+| External calendar event | A single session or an instance of recurring session that appears in a calendar. For example, an appointment or a class session. |
+| External calendar provider | The 3rd-party service that provides an external calendar. For example, [Google](https://developers.google.com/calendar/api/guides/overview), [Apple](https://developer.apple.com/documentation/foundation/calendar), or [Microsoft](https://learn.microsoft.com/en-us/graph/api/resources/calendar?view=graph-rest-1.0). |
+| External calendar resource | A resource that doesn't have a Wix Bookings calendar. Instead, the resource's availability is managed with an external calendar. For example, your app or the business owner may decide to manage availability through a staff member's Gmail account and Google calendar. |
+| Form | The booking form that's displayed to customers in the Wix site when they book the service. For example, customers can input their name into the bookings form. |
+| Integration | Extended functionality that's implemented by a 3rd-party service provider and made available to the merchant using Wix SPIs. For example, you, as a pricing provider, can offer custom varied pricing options to a Wix site using the [Wix Bookings Pricing Integration REST SPI](https://dev.wix.com/api/rest/wix-bookings/pricing-integration-spi). |
```

Comment:
Global change - we now say "service plugin" instead of SPI. 
For first time use in an article, we can say "service plugin (formerly SPI).